README.zh.md

```diff
@@ -0,0 +1,363 @@
+<p align="center" width="100%">
+<img src="https://i.postimg.cc/xTpWgq3L/pixiu-logo.png"  width="100%" height="100%">
+</p>
+
+<div>
+<div align="left">
+    <a target='_blank'>Qianqian Xie<sup>1</sup></span>&emsp;
+    <a target='_blank'>Weiguang Han<sup>1</sup></span>&emsp;
+    <a target='_blank'>Xiao Zhang<sup>2</sup></a>&emsp;
+    <a target='_blank'>Ruoyu Xiang<sup>7</sup></a>&emsp;
+    <a target='_blank'>Gang Hu<sup>4</sup></a>&emsp;
+    <a target='_blank'>Ke Qin<sup>4</sup></a>&emsp;
+    <a target='_blank'>Duanyu Feng<sup>3</sup></a>&emsp;
+    <a target='_blank'>Yongfu Dai<sup>3</sup></a>&emsp;
+    <a target='_blank'>Hao Wang<sup>3</sup></a>&emsp;
+    <a target='_blank'>Yanzhao Lai<sup>5</sup></a>&emsp;
+    <a target='_blank'>Min Peng<sup>1</sup></a>&emsp;
```

Comment:
Everytime you changed something in README.md, you should also notice that we should do the same change here